app.js

```diff
@@ -14,6 +14,22 @@
 
 'use strict';
 
+// eslint-disable-next-line import/order
+const AppConfig = require('./helpers/config.js');
+
+/**
+ * Default list of configuration options. Can be overridden by CLI.
+ *
+ * @memberof! app
+ * @default 'config.json'
+ */
+// eslint-disable-next-line import/order
+const appConfig = AppConfig(require('./package.json'));
+
+if (process.env.NEW_RELIC_LICENSE_KEY) {
```

Comment:
it'd be nicer to move this condition to `./helpers/newrelic_list` file and only initialize new relic if the environment variable is set. There is no need to have a conditional import here.

Reply:
In that case we have to do conditional `require('newrelic')` or have a file level conditional return. Because their pattern is as soon you require the file it initialized automatically. Initialization of newrelic means a big layer of wrappers on almost all native libraries. That I want to completely avoid. 

So I feel its the best way to keep this condition in that situation.